distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/CatalogBundle.java

```diff
@@ -210,14 +222,16 @@ public void setFanout(boolean fanoutEnabled) throws IOException {
     }
 
     public void setFanoutTagBlacklist(List<String> blacklist) throws IOException {
-        Map<String, Object> properties = adminConfig.getDdfConfigAdmin()
-                .getProperties(CATALOG_FRAMEWORK_PID);
+        Map<String, Object> properties = null;
+        try {
+            properties = Optional.ofNullable(adminConfig.getAdminConsoleService()
+                    .getProperties(CATALOG_FRAMEWORK_PID))
+                    .orElse(new Hashtable<>());
+        } catch (NotCompliantMBeanException e) {
+            //ignore
+        }
 
         if (blacklist != null) {
-            if (properties == null) {
-                properties = new Hashtable<>();
-            }
-
             properties.put("fanoutTagBlacklist", String.join(",", blacklist));
```

Comment:
❗️ `blacklist` may be null. We should put "" in in that case (or nothing at all)